client/reader/stream/index.jsx

```diff
@@ -133,6 +116,7 @@ class ReaderStream extends React.Component {
 		if ( this.props.shouldCombineCards ) {
 			items = combineCards( items );
 		}
+		items = map( items, this.props.transformStreamItems );
```

Comment:
i agree. I started with doing it first but the way that we currently apply injectRecommendation to initialize the `items` made it require enough goofing around that I hesitated.